core/ecschema-metadata/src/Metadata/Unit.ts

```diff
@@ -48,6 +48,19 @@ export class Unit extends SchemaItem {
   public get offset(): number { return this._offset; }
   public get denominator(): number { return this._denominator; }
 
+  /**
+   * Check if units phenomena's SchemaItemKey matches

```

Comment:
this is technically correct but it's probably more helpful if it said something like:
```
Returns true if a conversion can be calculated between the input units.
```

also make this method @alpha, I think it's going to be necessary for our public API